clang/test/CodeGenHLSL/builtins/select.hlsl

```diff
@@ -0,0 +1,45 @@
+// RUN: %clang_cc1 -finclude-default-header -x hlsl -triple \
```

Comment:
This might be a DXism that doesn't apply going forward with upstream, but

select is an HLSL 2021 and above feature. It doesn't seem like we have the proper gating if we can invoke it with out `-std=hlsl2021` https://devblogs.microsoft.com/directx/announcing-hlsl-2021/

Reply:
How do I fix this?

Reply:
nevermind, this isn't something the team is conserned about.